src/swapnaWJun20/ReverseNo.java

```diff
@@ -0,0 +1,20 @@
+package swapnaWJun20;
+
+public class ReverseNo {
+
+	//Reversing the number
+	void doReverse(int inp) {
+		int revNo = 0;
+		while (inp > 0) {
+			revNo = revNo * 10;
+			revNo = revNo + inp % 10;
+			inp = inp / 10;
+		}
```

Comment:
But, here integer is not reversed exactly - it is chopped digit by digit and printed in reverse order.
Please implement integer reverse logic - which is just a small extension to your login.
void findReverseNumber(int number) {
		int reverseNum = 0;
		int mod =0;
		while(number>0) {
			mod = number%10;
			number = number/10;
		    reverseNum = mod + reverseNum*10 ;
		}
		System.out.println("Reverse Number is "+reverseNum);
	}